pages/domains/domains_moving.md

```diff
@@ -30,7 +30,7 @@ Read more about each of these steps below.{.checklist}
 - [Plan to move your email to .gov](#plan-to-move-your-email-to-.gov)
 - [Get familiar with domain security best practices](#get-familiar-with-domain-security-best-practices)
 - [Develop a communications plan](#develop-a-communications-plan)
-- [Find out if you're eligible for financial assistance](#find-out-if-you're-eligible-for-financial-assistance)
+- [Find out if you’re eligible for financial assistance](#find-out-if-you‘re-eligible-for-financial-assistance)
```

Comment:
Unfortunately, the move to `’` from `'` means the ID becomes `#find-out-if-you%E2%80%99re-eligible-for-financial-assistance` and this link doesn't work anymore. 

😢 @gabydisarli

Reply:
Fixed

Reply:
Nevermind! @michelle-rago fixed this at [b91fda3](https://github.com/cisagov/getgov-home/pull/168/commits/b91fda3df24a2a6ed2b8ed0cb5c16f2c51dcb80e).

Reply:
@therealslimhsiehdy should I change these back to ' to make this work? I know we talked a lot about this...